scripts/sendpayment.py

```diff
@@ -65,6 +66,7 @@ def main():
     #without schedule file option, use the arguments to create a schedule
     #of a single transaction
     sweeping = False
+    bip79 = False
```

Comment:
I don't think bip79 is a good variable name here as what we are implementing aren't current version of BIP79.

Reply:
Fair point.